src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -312,6 +312,19 @@ const schema = {
             },
           },
         },
+        separationLocation: {
+          type: 'object',
+          properties: {
+            separationLocationCode: {
+              type: 'string'
```

Comment:
I was kind of surprised not to see a max length or pattern matching here. Are you simply confident that the FE will use the returned values from the endpoint response instead of ingesting user input?

Reply:
> Are you simply confident that the FE will use the returned values from the endpoint response instead of ingesting user input?

Yes, we won't let the user proceed unless the input exactly matches a predefined location.

Reply:
Oh, sorry... the "code" is predefined. It is provided when the "name" is set.